lib/pbench/cli/server/session_management.py

```diff
@@ -0,0 +1,182 @@
+import logging
+
+import click
+
+from pbench import BadConfig
+from pbench.cli.server import pass_cli_context
+from pbench.cli.server.options import common_options
+from pbench.server import PbenchServerConfig
+from pbench.server.auth.exceptions import KeycloakConnectionError
+from pbench.server.auth.keycloak_admin import Admin
+
+
+# Keycloak user session cli
+@click.group("user_sessions")
+@click.version_option()
+@pass_cli_context
+@common_options
+def user_command_cli(context):
+    # Entry point
+    pass
+
+
+@user_command_cli.command()
+@pass_cli_context
+@click.option(
+    "--user_id",
+    required=False,
+    help="Keycloak user id",
+)
+@click.option(
+    "--username",
+    required=False,
+    help="Keycloak username",
+)
+@click.option(
+    "--realm",
+    required=False,
+    help="Keycloak realm name",
+)
+@click.option(
+    "--admin_token",
+    required=True,
+    help="Keycloak realm name",
+)
+@common_options
+def get_user_sessions(context, user_id, username, realm, admin_token):
+    try:
+        logger = logging.getLogger(__name__)
+        config = PbenchServerConfig(context.config)
+        keycloak_admin = Admin(
+            server_url=config.get("keycloak", "server_url"),
+            realm_name="Master",
+            client_id="admin-cli",
+            logger=logger,
+            headers={"Authorization": f"Bearer {admin_token}"},
+        )
+        if user_id:
+            all_user_sessions = keycloak_admin.get_all_user_sessions(
+                user_id=user_id, realm=realm
```

Comment:
Please define `realm` somewhere, it will help to clear out things more.

Reply:
The file is no longer in the source tree.